src/dashboard/src/requirements/base.txt

```diff
@@ -9,7 +9,7 @@ django-extensions==1.1.1
 django-autoslug==1.7.1
 django-annoying==0.7.7
 elasticsearch>=1.0.0,<2.0.0
-git+https://github.com/artefactual/archivematica-fpr-admin.git@v1.3.0-beta.1#egg=archivematica-fpr-admin
+git+https://github.com/artefactual/archivematica-fpr-admin.git@dev/issue-9478-preforma#egg=archivematica-fpr-admin
```

Comment:
Now in qa/1.x we import archivematica-fpr-admin using pip. I've updated this using your branch.